api/src/test/java/com/code4ro/nextdoor/util/RandomObjectFiller.java

```diff
@@ -0,0 +1,81 @@
+package com.code4ro.nextdoor.util;
+
+import com.code4ro.nextdoor.core.entity.BaseEntity;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.RandomUtils;
+
+import java.lang.reflect.Field;
+import java.math.BigInteger;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.UUID;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.Size;
+
+public class RandomObjectFiller {
+
+    public static  <T> T createAndFill(Class<T> clazz) {
+        try {
```

Comment:
in loc de blocurile astea try/catch am putea folosi libraria https://www.vavr.io/ si aceste try/catch vor putea fi scrie in mod mai functional:
```java
Try.of(() -> ....).getOrElse(null);
```
sau ai putea sa returnezi direct Try<T> si sa verifici unde e folosit daca actiunea a avut loc cu succes sau nu

Reply:
Am incercat sa evit cat mai mult sa adaug librarii. Daca e cazul, cred ca se poate face mai tarziu.